src/matching/matcher.ts

```diff
@@ -17,11 +17,6 @@ export class PairMatcher {
     // useful for text objects.
     "<" : { match: ">",  nextMatchIsForward: true  },
     ">" : { match: "<",  nextMatchIsForward: false },
-    // These are useful for deleting closing and opening quotes, but don't seem to negatively
-    // affect how text objects such as `ci"` work, which was my worry.
-    '"' : { match: '"',  nextMatchIsForward: false  },
```

Comment:
Can you leave these inside PairMatcher? Because we use PairMatcher's pairings in other places, like surround.

Can you just give them an attribute like `directionless: true`? which you check for down below instead?